cmd/gitserver/server/commands.go

```diff
@@ -4,26 +4,26 @@ import (
 	"encoding/json"
 	"net/http"
 
-	"github.com/inconshreveable/log15"
-
 	"github.com/sourcegraph/sourcegraph/internal/gitserver/gitdomain"
 	"github.com/sourcegraph/sourcegraph/internal/gitserver/protocol"
+	"github.com/sourcegraph/sourcegraph/lib/log"
 )
 
 func handleGetObject(getObject gitdomain.GetObjectFunc) func(w http.ResponseWriter, r *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
 		var req protocol.GetObjectRequest
+		logger := log.Scoped("handleGetObject", "handles get object")
 
 		if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
 			http.Error(w, "decoding body", http.StatusBadRequest)
-			log15.Error("handleGetObject: decoding body", "error", err)
+			logger.Error("handleGetObject: decoding body", log.Error(err))
```

Comment:
If we are scoping the logger to "handleGetObject", we don't really need to add it again in the message. 

```suggestion
			logger.Error("decoding body", log.Error(err))
```